### PR TITLE
Added Ctrl+Click bypass.

### PR DESCRIPTION
**Pull request by jeffijoe**
Tested it in a local installation for my own project, works fine. Can't run build/tests cause I'm on Windows.

Fixes https://github.com/jmeas/backbone.intercept/issues/29

### CODE REVIEW

File: src/backbone.intercept.js [hunk 1]
```diff
@@ -60,6 +60,12 @@ Backbone.Intercept = {
       return;
     }
 
+    // If the Ctrl key is held, we want to open in a new tab.
+    bypass = e.ctrlKey;
+    if (bypass) {
```
Comment (jmeas) [inline src/backbone.intercept.js]: No need for an intermediary variable here, I think.

```js
if (e.ctrlKey) {}
```

**Comment by jmeas** [comment]
Thanks! I'll try to find time to do testing for this in the next few days.

**Comment by jeffijoe** [comment]
@jmeas still waiting :)

File: src/backbone.intercept.js (hunk 1 above)
Comment (jeffijoe) [inline src/backbone.intercept.js]: I am reusing the existing `bypass` variable, and I believe it clarifies what the check actually does.

**Comment by elgubenis** [comment]
lgtm, mind to add a test @jeffijoe?

**Comment by Haraldson** [comment]
Any progress here? And will it work with Macs, or should the code be expanded to look for `e.metaKey` as well?

**Comment by jmeas** [comment]
I really dropped the ball here. Sorry, y'all 😐 @jeffijoe / @Haraldson i added you both as collabs on the repo. Cutting a release for this now, though

**Comment by jeffijoe** [comment]
Would need to add `e.metaKey` as well, but I haven't been using Backbone for over a year so I'm not using this lib anymore. 😄

**Comment by jamesplease** [comment]
Published as 0.4.4!

**Pull request merged**